### DISCUSSION TITLE
Persuading Picture Agency to be consistent with Metadata fields used, and how useful information is delivered

### DISCUSSION THREAD

**Discussion by honorcb** [guardian/grid#3922] [General]
Is there a way  the Guardian and BBC can lobby agencies  in relation to additional metadata , and ways of identifying an agency "Picked image"  that  makes this information usable  in the Grid for the end users ,  without favouring an agency,

**Comment by honorcb** [comment]
How do we make agencies consistent on how they identify their "Picked" images ,   identify pictures that have "Embargo" dates ,  Kill notice images,  images that require updates to the metadata,  such as football match images listing the wrong event or teams.  

It is currently relevant because the IPTC is reviewing  RightML to see if there could be a few  standard messages that could accompany content that is both human and machine readable , giving a  basic standard between all the agencies.

**Comment by honorcb** [comment]
Our News teams have a request,   would it be possible to PA to give the Nation ,  where  the picture was taken ?
They use Getty because they can search for England,  or Scotland  to filter images,  but agencies appear not to do this.

They are looking for images such as those relating to the Weather in England,    which the agencies  group by Country , United Kingdom.

I know this is not simple,  as the group "location metadata"  fields have the cardinality of 0 ..1   so cannot have United Kingdom and England in the Country field

**Reply by paperboyo** [reply]
Interesting. I have looked into 2 images from both PA and Getty. None of them are using newest Location Shown (Grid doesn’t support it, but last time I checked nobody is providing it anyway). They both use legacy IPTC and XMP versions of Location data. 

Relevant exiftool output below:

PA:
```
IPTC.Country-PrimaryLocationName = UK
XMP.photoshop:Country = UK
```
(no state info)

Grid displays:
![image](https://user-images.githubusercontent.com/6032869/213592365-2aa16b20-4514-425a-ac48-1dd7c7c89318.png)

Getty:
```
IPTC.Country-PrimaryLocationName = United Kingdom
IPTC.Country-PrimaryLocationCode = GBR
XMP.photoshop:Country = United Kingdom
XMP.Iptc4xmpCore:CountryCode = GBR
IPTC.Province-State = England
XMP.photoshop:State = England
```

Grid displays:
![image](https://user-images.githubusercontent.com/6032869/213592829-4200f2a1-6a8c-4ec9-88bc-fbedfb5a44a8.png)

So Getty is using [State/Province legacy](https://www.iptc.org/std/photometadata/specification/IPTC-PhotoMetadata#province-or-state-legacy) field to express England. It’s unclear to me if that’s correct from the standards POV.

But it’s only Getty that seems to be using State for United Kingdom nations consistently. All other major agencies I checked (AP, AAP, EPA, Reuters and indeed PA) do it one time in a thousand.

Not sure how likely is it to convince all of them to do it consistently. Even more so, because it’s not clear it’s even correct use of State/Province field. Plus, it’s “legacy” for a reason. Looking at the newer Location structure, it’s also not clear what [State/Province](https://www.iptc.org/std/photometadata/specification/IPTC-PhotoMetadata#province-or-state) should contain. It’s clear Country [should be United Kingdom](https://www.iso.org/obp/ui/#iso:code:3166:GB), but that’s about it. Quick Googling and trawl of IPTC group doesn’t give any useful answers either… Convincing agencies to strictly stick to standards seems hard enough, asking them to follow conventions may be harder, I fear.

**Comment by honorcb** [comment]
https://iptc.org/std/photometadata/specification/IPTC-PhotoMetadata#location-structure
is I think  most recent essay about this, 

Getty use of State Province does not allow them to stay which "county" the picture was taken in,  and I know for PA  regional customers that more important than the Nation,
